scripts/update_VM.sh

```diff
@@ -1,10 +1,25 @@
 #!/bin/sh
 
 echo "Updating..."
+SIRF_TAG="default"
```

Comment:
this will lead to repetition. Much better to just pass all arguments. Say below
```bash
   ./UPDATE.sh $*
```

Reply:
good idea